src/components/UI/NavBar/dropdown.js

```diff
@@ -0,0 +1,33 @@
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import './dropdown.scss';
+import { logout } from '../../../Redux/actions/actionCreators/auth';
+
+const Dropdown = () => {
+  const dispatch = useDispatch();
+  const logOut = () => {
+    dispatch(logout());
+  };
+  return (
+    <>
+      <ul className='dropdown-menu'>
+      <li className='dropdown-item'>
+              Pesto
```

Comment:
These are hardcoded for testing purpose , values will be fetched from state upon updation of backend logic